<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDates</class>
 <widget class="QDialog" name="EditDates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Dates Label - %1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Render area</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="posX">
            <property name="minimum">
             <number>-2147483647</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="posY">
            <property name="minimum">
             <number>-2147483647</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="width">
            <property name="minimum">
             <number>-2147483647</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="height">
            <property name="minimum">
             <number>-2147483647</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Labels colour</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Weakdays</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="colWeakdayHex">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colWeakdayPreview">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectWeakdayCol">
          <property name="text">
           <string>Select colour</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Weakends</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="colWeakendHex">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colWeakendPreview">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectWeakendCol">
          <property name="text">
           <string>Select colour</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Weakstrats</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="colWeakstartHex">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colWeakstartPreview">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectWeakstartCol">
          <property name="text">
           <string>Select colour</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Text</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Font</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fontPreview">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Select font</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Text alignment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="textAlign">
          <item>
           <property name="text">
            <string>Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Center</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Vertical</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Date markers</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTreeWidget" name="markers">
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>175</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Markers group/Event name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Colour/Date</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addItem">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addChild">
          <property name="text">
           <string>-L</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeItem">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>posX</tabstop>
  <tabstop>posY</tabstop>
  <tabstop>width</tabstop>
  <tabstop>height</tabstop>
  <tabstop>colWeakdayHex</tabstop>
  <tabstop>selectWeakdayCol</tabstop>
  <tabstop>colWeakendHex</tabstop>
  <tabstop>selectWeakendCol</tabstop>
  <tabstop>colWeakstartHex</tabstop>
  <tabstop>selectWeakstartCol</tabstop>
  <tabstop>fontPreview</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>textAlign</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>markers</tabstop>
  <tabstop>addItem</tabstop>
  <tabstop>addChild</tabstop>
  <tabstop>removeItem</tabstop>
  <tabstop>ok</tabstop>
  <tabstop>cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>selectWeakstartCol</sender>
   <signal>clicked()</signal>
   <receiver>EditDates</receiver>
   <slot>onSelectWeakstartColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectWeakendCol</sender>
   <signal>clicked()</signal>
   <receiver>EditDates</receiver>
   <slot>onSelectWeakendColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addChild</sender>
   <signal>clicked()</signal>
   <receiver>EditDates</receiver>
   <slot>onAddChild()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addItem</sender>
   <signal>clicked()</signal>
   <receiver>EditDates</receiver>
   <slot>onAddItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>EditDates</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>EditDates</receiver>
   <slot>onAccepted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeItem</sender>
   <signal>clicked()</signal>
   <receiver>EditDates</receiver>
   <slot>onRemoveItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectWeakdayCol</sender>
   <signal>clicked()</signal>
   <receiver>EditDates</receiver>
   <slot>onSelectWeakdayColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditDates</receiver>
   <slot>onSelectFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAccepted()</slot>
  <slot>onAddItem()</slot>
  <slot>onAddChild()</slot>
  <slot>onRemoveItem()</slot>
  <slot>onSelectWeakdayColour()</slot>
  <slot>onSelectWeakendColour()</slot>
  <slot>onSelectWeakstartColour()</slot>
  <slot>onSelectFont()</slot>
 </slots>
</ui>
